ICAFIX/ApplyAutoReclean.sh

```diff
@@ -236,7 +249,15 @@ for fMRIName in ${fMRINameToUse} ; do
         --reclassify_as_noise_file="$ReclassifyAsNoiseTxt"
     )
 
+    if [ "$UseLocalPython" = "FALSE" ]; then
+        # use singularity
+        PythonLaunchCommand=("${singularity_command[@]}" "${pythonCode[@]}")
+    else
+        # use native env
+        PythonLaunchCommand=("${HCPCONDAENV}/bin/python3" "${pythonCode[@]}")
```

Comment:
I would expect the user to have already done "conda activate <whatever>" before starting this pipeline.  Do we really need a special `HCPCONDAENV` variable, rather than using whatever python3 is on `PATH`?  Alternatively, how about a pipeline parameter of `--python-interpreter` that defaults to a bare `python3`, and the user can just point at a special python if they need to?

Reply:
During the implementation, I found that the default python is always the fslconda on our end, which will never work for this script because of the lack of proper modules. I think the `--python-interpreter` is a good idea - will add it in next commit.

Reply:
That may be due to how we modify `PATH` in the `SetUp...` stuff.  We can check for `python3` before we modify `PATH`, and then add its directory back onto the beginning.  But, the user may have already put the fsl stuff onto the start of their PATH, so...